sdk/eventhub/azure-eventhub/stress/dev_requirement.txt

```diff
@@ -2,6 +2,7 @@ psutil
 ..
 ../../azure-eventhub-checkpointstoreblob
 ../../azure-eventhub-checkpointstoreblob-aio
-azure-servicebus
+azure-servicebus==0.50.3
 azure-storage-blob
 azure-identity
+opencensus-ext-azure
```

Comment:
is `opencensus.stats` part of the same package?

Reply:
yep, opencensus-ext-azure depends on opencensus